vim.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Vim (Vi IMproved)
+Version = 8.2.5171
+License = GPL-compatible charityware
+LicenseType = 1
+Description = Vim is a highly configurable text editor built to enable efficient text editing. It is an improved version of the vi editor distributed with most UNIX systems.
+Category = 7
+URLSite = http://www.vim.org/
+URLDownload = https://github.com/vim/vim-win32-installer/releases/download/v8.2.5171/gvim_8.2.5171_x86_signed.exe
+SHA1 = 40b579d4ea767a533524256a289e4e14eb8a4c81
+SizeBytes = 9841432
+
```

Comment:
```suggestion
Icon = vim.ico

```

Reply:
The text under [Icon](https://reactos.org/wiki/RAPPS#Icon) in the wiki reads:

> if the filename of the .txt file matches the filename of an .ico file, that icon will also be displayed.

Suggesting that the addition of the field would technically be a pointless declaration. I did try placing the file in `appdb/` and it did show up before pushing the change. Furthermore, out of the currently existing `.ico` files of this repository; 11 do not share their basename with any `.txt` file; of those who do, 41 rely on the filename match rule and 23 explicitly add the _Icon_ property. I did not analyze the history. Is there a preference in moving towards specifying the icon file purely for documentation purposes?

Reply:
There is no preference currently, so omitting it is perfectly fine

Reply:
Didn't knew that, it's alright then. 🙂